client/components/Project/Project.js

```diff
@@ -61,7 +61,7 @@ class Project extends Component {
     return (
       <div className={projectClasses}
         onClick={this.handleClick.bind(this)}>
-        <img className={styles.image} src={project.image || require('../../images/logo.png')} />
+        <img className={styles.image} src={project.imageUrl || require('../../images/logo.png')} />
```

Comment:
require('../../images/logo.png') <-- this should be an import at the beginning of the file, such as:
 - import logo from '../../images/logo.png'.
Then you just need to do project.imageUrl || logo.